Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+"nodePubKey": "02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6",
+
+"address": "xud2.test.exchangeunion.com:8885",
+"nodePubKey": "028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a"
+
+"address": "xud3.test.exchangeunion.com:8885",
+"nodePubKey": "03fd337659e99e628d0487e4f87acf93e353db06f754dccc402f2de1b857a319d0"
+```
+You already received existing orders from the network. You can view these with
+```
+xucli getorders LTC/BTC
+```
+Let's execute a test order to trigger a match and execution via atomic swap by e.g. buying 0.0005 litecoin with bitcoin
+```
+xucli buy 0.0005 LTC/BTC 1.13
+```
+which should result in sth. like this:
+```
+matched 0.0001 BTC @ 1.13 with order da8d3401-e75a-11e8-972d-a9036be5a943
+No more matches found
+```
+
+Currently the trading bots of xud1, 2 & 3 are configured to simply replace successfully filled and swapped orders. A more realistic, market dynamic behavior will be deployed soon.
+
+### API Integration
+`xud`'s gRPC API overview can be found [here](https://exchangeunion.github.io/xud-typedoc/classes/_grpc_grpcservice_.grpcservice.html).
+
+From a high-level perspective, `xud` supports two modes:
+1. `matching`
+2. `nomatching`
+
+Default is `matching` mode, which means that `xud` acts as order book & matching engine for all of your orders. Choose this mode if your exchange is not live yet, comparably small or you simply want to rather rely on the battle-tested matching engine of `xud` instead of using your own. You will need to design your exchange in a way to add orders to `xud`'s order book and listen to order events, in order to display real-time order information to your users and update user account balances. The following API calls are relevant:
+* `subscribeAddedOrders` this streaming call subscribes to orders being added to the order book. Together with `SubscribeRemovedOrders`, this allows the client to maintain an up-to-date view of the order book. Use this call to show a real-time order book to users via your exchange front-end and update account balances for your users.
+* `SubscribeRemovedOrders` this streaming call subscribes to orders being  removed - either in full or in part - from the order book. Together with `SubscribeAddedOrders`, this allows the client to maintain an up-to-date view of the order book. Use this call to show a real-time order book to users via your exchange front-end and update account balances for your users.
+* `placeOrder` use this to add an order to `xud`'s order book, which was triggered by a user of your platform. You can choose the sync version of the call, which returns the final result. This can take some seconds if the order gets swapped with a remote peer. The async version updates with events in the course of the swap via `subscribeSwaps`. Currently buy/sell as market/limit order are supported. 
+* `cancelOrder` cancel an order previously placed via `placeOrder`, which is triggered by a user of your platform.
```

Comment:
It's `RemoveOrder` now, we renamed the call.

Reply:
Oh, done.